srcpkgs/wine/template

```diff
@@ -61,20 +67,40 @@ replaces="libwine>=0"
 # This testsuite might hang indefinitely
 make_check=extended
 
+case "$XBPS_MACHINE" in
+	aarch64*) no_cross=yes # TODO: testing
+esac
+
+
 if [ "$XBPS_TARGET_WORDSIZE" != 32 ]; then
-	configure_args+=" --enable-win64 --libdir=/usr/lib"
+	_64bit_configure_args=" --enable-win64 --libdir=/usr/lib"
+	configure_args+=_64bit_configure_args
 	_wine_suffix="64"
 	_wineserver_suffix=${_wine_suffix}
 else
 	_wineserver_suffix="32"
 	CFLAGS=" -D_LARGE_FILE_SOURCE=1 -D_FILE_OFFSET_BITS=64"
 fi
 
+if [ "$CROSS_BUILD" ] && [ $(vopt_if mingw true false) ]; then
```

Comment:
```suggestion
if [ "$CROSS_BUILD" ] && [ "$build_option_mingw" ]; then
```
pretty sure what you had would always evaluate true

also, put the conditional inside the function

Reply:
Mingw should be reversed 
`if [ "$CROSS_BUILD" ] && [ $(vopt_if mingw false true) ]; then`
then
`if [ "$CROSS_BUILD" ] && [ ! "$build_option_mingw" ]; then`

Reply:
that's not how `test` works, you need to use `-z` for that